pkgs/by-name/cr/crow/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  runCommand,
+  fetchFromGitHub,
+  cmake,
+  asio,
+  python3,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "crow";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "crowcpp";
+    repo = "crow";
+    rev = "v${version}";
+    sha256 = "sha256-fokj+KiS6frPVOoOvETxW3ue95kCcYhdhOlN3efzBd4=";
+  };
+
+  propagatedBuildInputs = [ asio ];
+  nativeBuildInputs = [
+    asio
+    cmake
+    python3
+  ];
+
+  cmakeFlags = [
+    "-DCROW_BUILD_EXAMPLES=OFF"
```

```suggestion
    (lib.cmakeBool "CROW_BUILD_EXAMPLES" true)
```

Docs: https://noogle.dev/f/lib/cmakeBool

---

the more you know :+1: updated (fyi: off translates to false :sweat_smile:)